schoolmatch.py

```diff
@@ -13,152 +20,171 @@
 import pandas as pd
 import numpy as np
 
-OSPI_df = pd.read_csv('Washington_School_Directory_20171121.csv')
-OSPI_Ind_df = pd.read_csv('ApprovedPrivateSchoolsList.csv')
 
-RS_df_pre = pd.read_csv('RSSchoolList2.csv', usecols=[0,1,2,3,4])
-RS_df_NotPub = RS_df_pre[RS_df_pre.Type != "Public"]
-RS_df = RS_df_pre[RS_df_pre.Type == "Public"]
+def clean_title(text):
+    """
+    Takes a pandas series of strings, and returns a pandas series of strings
+    that have been stripped and each word capitalized
+    """
+    text = text.str.lower()
+    text = text.str.title()
+    text = text.str.strip()
+    return text
+
+
+def string_truncate(strings_to_truncate, words_to_remove, truncated_list_name):

```

Comment:
Ok, right. Because later I call it by binding it to a variable, the value of the locally defined truncated_list_name gets added to my globally defined ones later. I think this will get clearer as I write more programs with fewer global variables.

Reply:
Within this function, I do define truncated_list_name as an empty list that I'll append as I loop through and modify school names. Is there a cleverer or more pythonic way of doing this, or is defining the empty list then appending a good enough way?